### PR TITLE
Update fritzing.desktop

### PR DESCRIPTION
**Pull request by glixx**
GenericName must not be same with Name.
Fix wrong icon name.

### CODE REVIEW

**Comment by glixx** [comment]
Fix wrong icon name

File: fritzing.desktop [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-Comment=Electronic Design Automation software
+GenericName=Electronic Design Automation software
+GenericName[fi]=Elektroniikan suunnitteluohjelma
+GenericName[ru]=Программа проектирования электроники
+Comment=Create circuits and lay out printed circuit boards
+Comment[fi]=Suunnittele elektronisia kytkentöjä ja piirilevyjä
+Comment[ru]=Создавайте схемы и выкладывайте печатные платы
 Exec=Fritzing
-Icon=fritzing_icon
+Icon=fritzing
```
Comment (ovidiub13) [inline fritzing.desktop]: the name of the file is `fritzing_icon.png`. Why this change?
